docs/features/network-security-controls/network-policy.md

```diff
@@ -1,25 +1,141 @@
 # NetworkPolicy
 
+## Introduction
+
+Network Policy is a new resource, which helps enforce the network isolation of pods. By default, the network traffic from and to K8s pods are not restricted in any way: NetworkPolicy resources allow us to enable and/or disable specific dataflows at a pod level.
```

Comment:
I am not sure if Network Policy is a "new" resource, it has been present in k8s for ages